tools/verify-fs.in

```diff
@@ -95,25 +95,36 @@ for ROOTFS in *.squashfs; do
     fi
 
     if [ -s "${ROOTFS}.sha256" ] ; then
-        if [ /usr/bin/sha256sum -c "${ROOTFS}.sha256" 2>/dev/null ] ; then
-            log "Good integrity of ${ROOTFS}, SHA256 checksum is valid"
+        if /usr/bin/sha256sum -c "${ROOTFS}.sha256" 2>/dev/null; then
+            if [ -s "${ROOTFS}-manifest.json" ] ; then
+                    SHA256_COMPUTED=$(cut -d ' ' -f1 < "${ROOTFS}.sha256")
+                    SHA256_MANIFEST=$(/usr/share/fty/scripts/JSON.sh -l < "${ROOTFS}-manifest.json" | grep '\["application","ShaChecksum"\]' | cut -f2 | cut -d '"' -f2)
```

Comment:
Note: JSON processing here can be much streamlined and simplified (for consumer call syntax) by using the updated tool version from master's PR #408